.gitignore

```diff
@@ -6,10 +6,13 @@ __pycache__/
 ros2_ws/build/
 ros2_ws/install/
 ros2_ws/log/
+ignore/
```

Comment:
Er is (nog) geen ignore-directory. Wordt die aangemaakt tijdens runtime toevallig?

Reply:
Deze is op verzoek van Yuri, om af en toe even makkelijk een mapje ergens aan te kunnen maken voor wat tijdelijke bestanden, die je niet wil pushen. We hadden afgesproken dat hiervoor een map genaamd ignore gebruikt kan worden, wat nu dus overal in de repo ignored wordt.